flink-yarn/src/test/java/org/apache/flink/yarn/YarnClusterDescriptorTest.java

```diff
@@ -115,6 +113,7 @@ public void testFailIfTaskSlotsHigherThanMaxVcores() throws ClusterDeploymentExc
 
 			fail("The deploy call should have failed.");
 		} catch (ClusterDeploymentException e) {
+			e.printStackTrace();
```

Comment:
Why do we need `e.printStackTrace()`?

Reply:
Nice catch. That was for debugging and I forgot to remove it.